### DISCUSSION TITLE
Automa tutorial needed

### DISCUSSION THREAD

**Discussion by WaleedThePro** [AutomaApp/automa#165] [Q&A]
Is there anywhere I can find out how to use it, wiki seems empty and no tutorials on youtube. I would really appreciate any information a reddit or discord would be appreciated as well.
I wanted to automate a new tab and pin the tab once it's open. i can't figure out how to pin the tab

**Comment by MagEpub** [comment]
I started one, but I'm just a beginner myself

https://github.com/MagEpub/Open

The single best source I found was his silent 1m 34s youtube video

12/11/2021: Created a procedure for how to create a simple workflow, with pictures.

**Reply by MagEpub** [reply]
Did a brief one on the element selector.  Beginner level.

https://github.com/MagEpub/Open2/blob/snippet1/README.md
